scripts/download-tools.ts

```diff
@@ -267,7 +267,7 @@ async function downloadGDB(target: VsceTarget, dest: string, options?: ToolOptio
 
     const fileType  = await fileTypeFromFile(downloadFilePath);
 
-    if (nodeOs.platform() === 'win32') {
+    if (fileType?.mime === 'application/zip' || fileType?.ext === 'zip') {
```

Comment:
Another change here, @JonatanAntoni . Realized we run all packaging/repackaging on Linux.

Reply:
See later comment what fixed it.